<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <author>BBDXF</author>
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>586</width>
    <height>308</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(85, 85, 127);
color: rgb(255, 255, 255);
</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>9</x>
      <y>9</y>
      <width>570</width>
      <height>292</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="layout_up">
     <item>
      <layout class="QVBoxLayout" name="layout_left">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="QLabel" name="label_main_title">
           <property name="text">
            <string>PQ音乐播放器</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_main_ver">
           <property name="text">
            <string>V 1.0.1</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout">
         <property name="spacing">
          <number>0</number>
         </property>
         <item>
          <widget class="QLabel" name="label_title">
           <property name="text">
            <string>刘德华</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="Phonon::SeekSlider" name="slider_ctr_progree">
           <property name="maximumSize">
            <size>
             <width>16777215</width>
             <height>15</height>
            </size>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_time">
           <property name="layoutDirection">
            <enum>Qt::LeftToRight</enum>
           </property>
           <property name="text">
            <string>00:00/00:00</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QCheckBox" name="checkBox_lrc">
           <property name="text">
            <string>歌词</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QToolButton" name="toolButton_left">
           <property name="text">
            <string>&lt;</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QToolButton" name="toolButton_play">
           <property name="text">
            <string>||</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QToolButton" name="toolButton_right">
           <property name="text">
            <string>&gt;</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="Phonon::VolumeSlider" name="slider_volume">
           <property name="maximumSize">
            <size>
             <width>16777215</width>
             <height>15</height>
            </size>
           </property>
           <property name="iconSize">
            <size>
             <width>15</width>
             <height>15</height>
            </size>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QListView" name="listView"/>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="layout_right">
       <item>
        <layout class="QHBoxLayout" name="layout_search">
         <item>
          <widget class="QLineEdit" name="lineEdit_search"/>
         </item>
         <item>
          <widget class="QToolButton" name="toolButton_baidu">
           <property name="text">
            <string>百度音乐</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QToolButton" name="toolButton_163">
           <property name="text">
            <string>网易云音乐</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QListView" name="listView_2"/>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>Phonon::SeekSlider</class>
   <extends>QWidget</extends>
   <header location="global">phonon/seekslider.h</header>
  </customwidget>
  <customwidget>
   <class>Phonon::VolumeSlider</class>
   <extends>QWidget</extends>
   <header location="global">phonon/volumeslider.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
